### DISCUSSION TITLE
How to use Network ?

### DISCUSSION THREAD

**Discussion by RetroAndDev** [CosmosOS/Cosmos#1642] [Q&A]
Hello, i see the network drivers has been implemented, but how i can use ? I’m on the lasset compiled sources.
Thanks for your help

**Comment by quajak** [comment]
Cosmos currently does not have any part of the network stack currently implemented, so you would have to implement it yourself.
https://github.com/CosmosOS/Cosmos/pull/1633 adds quiet a large part of it but further work is required.

**Comment by RetroAndDev** [comment]
Ok, if i implement the network of aura, how i can use it ?

**Reply by quajak** [reply]
Networking isnt easy and I will have to read up on the network stack https://wiki.osdev.org/Network_Stack and compare that with what is implemented by the PR to understand what is already supported and what you have to work on yourself. Without understanding the code/networking in general, you wont be able to do it.

**Comment by valentinbreiz** [comment]
hi @ArtGo-Devloppper, #1633 will be merged soon. You will be able to use UDP, DNS, DHCP... ;)

**Reply by ghost** [reply]
:)
